src/widgets/equalizerslider.h

```diff
@@ -19,6 +19,7 @@
 #define EQUALISERSLIDER_H
 
 #include <QWidget>
+#include <QFontMetrics>
```

Comment:
Seems like you need this include only in the cpp file, so please move it there: it's better to include files only where necessary.